src/sqlancer/doris/gen/DorisNewExpressionGenerator.java

```diff
@@ -320,12 +327,29 @@ public DorisExpression generateConstant(DorisDataType type) {
         return createConstant(type);
     }
 
+    public DorisExpression createConstantWithoutNull(DorisDataType type) {
+        DorisExpression constant = createConstant(type);
+        int loopCount = 0;
+        while (constant instanceof DorisConstant.DorisNullConstant && loopCount < 1000) {
```

Comment:
This is fine for now, but we could also consider extracting a common utility method that attempts an action for a number of times and throws an `IgnoreMeException` if things fail.